submissions/Valerii49/side_menu/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="shortcut icon" href="favicon.ico">
+	<link rel="apple-touch-icon" href="favicon.png">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+	<title>DOM</title>
+</head>
+
+<body>
+	<div class="wrapper">
+		<header class="header">
+			<div class="header__container">
+				<a href="#" class="header__logo"><img src="img/logo.png" alt="Logotipe">
+					<p>Kottans Front-End Courses</p>
+				</a>
+				<div class="header__title">7. Document Object Model practice</div>
+				<div class="menu__icon icon-menu">
+					<div class="arrow-main">
+						<div></div>
+					</div>
+					<span></span>
```

Comment:
same about span

Reply:
i need <span> for animation Burger Icon. i can't use <div class="arrow-main">